ansible/ansible.cfg

```diff
@@ -14,7 +14,6 @@ squash_actions      = apk
 # since we have all information available.
 [ssh_connection]
 ssh_args            = -F /dev/null -o ControlMaster=auto -o ControlPersist=60s
-scp_if_ssh          = True
```

Comment:
This will affect all platforms right?  Seem like what we need for z/OS but not sure if we want the same for all platforms?

Reply:
I haven't found a way to unset this if set.

Reply:
Found a setting (`ssh_transfer_method`) so reverted this back to True.